### PR TITLE
v0.9.0

### PR DESCRIPTION
**Pull request by 0xCAB0**
In an effort to follow the mono-repo principle we are trying to adopt at Thalos, we decided to "de-modularize" the scrapping part of the project ( previously located in [this repo](https://github.com/ThalosES/vinted-db-feeder) ) and integrate it as part of the vinted-rs code base.

This may not have any impact so far in the Rust functionality, however, it should be consider a major change.

### CODE REVIEW

**Comment by 0xCAB0** [comment]
BTW, sorry for the inconvenience, I switched to NIX and forgot to export my PGP key for signing, I will fix it soon.

File: scrapping/vinted-db-feeder/lib/get_categories.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        self.parent_id = parent_id
+        self.url = url
+        self.url_en = url_en
+        self.children_ids = children_ids
+        self.children = []
+
+    def __str__(self):
+        return f"{self.id},'{self.title}','{self.code}',{self.parent_id},'{self.url}','{self.url_en}'"
+
+    @staticmethod
```
Comment (pxp9) [inline scrapping/vinted-db-feeder/lib/get_categories.py:18]: why this if you have arg self ?

File: scrapping/vinted-db-feeder/lib/get_sizes.py [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    def __init__(self, id, title, size_type):
+        self.id = id
+        self.title = title
+        self.category = size_type
+    
+    def __str__(self):
+        return f"{self.id}, '{self.title}', '{self.category}'"
+
+
+def obtener(path):
```
Comment (pxp9) [inline scrapping/vinted-db-feeder/lib/get_sizes.py:23]: english plz

**Review by pxp9 — CHANGES_REQUESTED** [review]
just a few comments, nice work

File: scrapping/vinted-db-feeder/lib/get_sizes.py (hunk 2 above)
Comment (0xCAB0) [reply scrapping/vinted-db-feeder/lib/get_sizes.py]: Done in the last commit

File: scrapping/vinted-db-feeder/lib/get_categories.py (hunk 1 above)
Comment (0xCAB0) [reply scrapping/vinted-db-feeder/lib/get_categories.py]: NVM, it was the @staticmethod, fixed it in the last commit.

**Review by pxp9 — APPROVED** [review]
Lgtm !

**Pull request merged**